borgmatic/config/schema.yaml

```diff
@@ -157,3 +157,52 @@ map:
                 desc: Restrict the number of checked archives to the last n. Applies only to the
                       "archives" check.
                 example: 3
+    hooks:
+        desc: |
+            Shell commands or scripts to execute before and after a backup or if an error has occurred.
+            IMPORTANT: All provided commands and scripts are executed with user permissions of borgmatic.
+            Do not forget to set secure permissions on this file as well as on any script listed (chmod 0600) to
```

Comment:
Should the scripts themselves be 0700? Or will `check_call()` execute them regardless of whether they're marked executable?

Reply:
My bad! 0700 is correct, 0600 would give "permission denied", just as with an ordinary shell.

Reply:
Although you are correct here that 0600 works for the borgmatic config file itself. That doesn't need to be executable, of course.